apps/studio/components/interfaces/SQLEditor/UtilityPanel/ChartConfig.tsx

```diff
@@ -0,0 +1,176 @@
+      <>
+        <ResizablePanel className="p-4" defaultSize={85}>
+          <div className="h-full">
+            {config.type === 'bar' && (
+              <BarChart
+                showLegend
+                size="normal"
+                xAxisIsDate={xKeyDateFormat === 'date'}
+                data={resultToRender as any}
+                xAxisKey={config.xKey}
+                yAxisKey={config.yKey}
+                emptyStateMessage="Execute a query and configure the chart options"
+              />
+            )}
+          </div>
+        </ResizablePanel>
+        <ResizableHandle withHandle />
+        <ResizablePanel defaultSize={15} minSize={15} className="px-3 py-3 space-y-4">
+          <>
+            <div className="flex justify-between items-center h-5">
+              <h2 className="text-sm text-foreground-lighter">Chart options</h2>
+              {config.xKey && config.yKey && (
+                <Button
+                  type="text"
+                  onClick={() => {
+                    const currentX = config.xKey
+                    const currentY = config.yKey
+                    onConfigChange({ ...config, xKey: currentY, yKey: currentX })
+                  }}
+                  title="Swap X and Y axis"
+                  icon={<ArrowUpDown size="15" className="text-foreground-lighter" />}
+                >
+                  Flip
+                </Button>
+              )}
+            </div>
+
+            <Select_Shadcn_
+              value={config.xKey}
+              onValueChange={(value) => {
+                onConfigChange({ ...config, xKey: value })
+              }}
+            >
+              <SelectTrigger_Shadcn_>
+                X Axis {config.xKey && `- ${config.xKey}`}
+              </SelectTrigger_Shadcn_>
+              <SelectContent_Shadcn_>
+                <SelectGroup_Shadcn_>
+                  {resultKeys.map((key) => (
+                    <SelectItem_Shadcn_ value={key} key={key}>
+                      {key}
+                    </SelectItem_Shadcn_>
+                  ))}
+                </SelectGroup_Shadcn_>
+              </SelectContent_Shadcn_>
+            </Select_Shadcn_>
+            <Select_Shadcn_
+              value={config.yKey}
+              onValueChange={(value) => {
+                onConfigChange({ ...config, yKey: value })
+              }}
+            >
+              <SelectTrigger_Shadcn_>
+                Y Axis {config.yKey && `- ${config.yKey}`}
+              </SelectTrigger_Shadcn_>
+              <SelectContent_Shadcn_>
+                <SelectGroup_Shadcn_>
+                  {resultKeys.map((key) => (
+                    <SelectItem_Shadcn_ value={key} key={key}>
+                      {key}
+                    </SelectItem_Shadcn_>
+                  ))}
+                </SelectGroup_Shadcn_>
+              </SelectContent_Shadcn_>
+            </Select_Shadcn_>
+
+            <Checkbox
```

Comment:
use Checkbox_shadcn ?

This is a change I think is worth doing to reduce refactor later.